### PR TITLE
Medium

### PR DESCRIPTION
**Pull request by andrisLaduzans**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -8,4 +8,4 @@ steps:
   - script: |
       docker build -t javascript-exercises:latest .
```
Comment (sandris-) [inline azure-pipelines.yml]: where is prettier:check?

File: exercises/medium/ScrabbleScore/index.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+// import { ModuleMap } from "jest-haste-map";
```
Comment (sandris-) [inline exercises/medium/ScrabbleScore/index.js]: is this needed?

File: azure-pipelines.yml (hunk 1 above)
Comment (andrisLaduzans) [reply azure-pipelines.yml]: fixed

File: exercises/medium/ScrabbleScore/index.js (hunk 2 above)
Comment (andrisLaduzans) [reply exercises/medium/ScrabbleScore/index.js]: no, fixed

**Pull request merged**